sklearn/neighbors/_unsupervised.py

```diff
@@ -83,32 +93,30 @@ class NearestNeighbors(NeighborsBase, KNeighborsMixin,
       >>> neigh.kneighbors([[0, 0, 1.3]], 2, return_distance=False)
       array([[2, 0]]...)
 
-      >>> nbrs = neigh.radius_neighbors([[0, 0, 1.3]], 0.4, return_distance=False)
+      >>> X = [[0, 0, 1.3]]
+      >>> nbrs = neigh.radius_neighbors(X, 0.4, return_distance=False)
       >>> np.asarray(nbrs[0][0])
       array(2)
-
-    See also
-    --------
-    KNeighborsClassifier
-    RadiusNeighborsClassifier
-    KNeighborsRegressor
-    RadiusNeighborsRegressor
-    BallTree
-
-    Notes
-    -----
-    See :ref:`Nearest Neighbors <neighbors>` in the online documentation
-    for a discussion of the choice of ``algorithm`` and ``leaf_size``.
-
-    https://en.wikipedia.org/wiki/K-nearest_neighbor_algorithm
     """
 
-    def __init__(self, n_neighbors=5, radius=1.0,
-                 algorithm='auto', leaf_size=30, metric='minkowski',
-                 p=2, metric_params=None, n_jobs=None, **kwargs):
+    def __init__(
+        self,
+        n_neighbors=5,
+        radius=1.0,
+        algorithm="auto",
+        leaf_size=30,
+        metric="minkowski",
+        p=2,
+        metric_params=None,
+        n_jobs=None,
+    ):
```

Comment:
Please revert this changes and also keep the "kwargs". That should be addressed in a separate PR as it's technically an API change (see the issue you opened).

Reply:
Understood and reverted, thanks.